app/javascript/components/editor/TestRunOutput.tsx

```diff
@@ -8,7 +8,12 @@ export const TestRunOutput = ({
 }: {
   testRun: TestRun
 }): JSX.Element => {
-  if (testRun.version >= 3 && testRun.tasks.length > 0) {
+  const hasTasks =
+    testRun.version >= 3 &&
+    testRun.tasks.length > 0 &&
+    testRun.tests.filter((t) => t.taskId == null || t.taskId == undefined) == 0
```

Comment:
Funny thing: this is testing whether an empty array is equal to `0`, which funnily enough, it is in JS! I'll refactor next week.